bin/dfx-software-idl2json-install

```diff
@@ -2,17 +2,20 @@
 set -euo pipefail
 SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
 PATH="$SOURCE_DIR:$PATH"
+
 . "$SOURCE_DIR/versions.bash"
+PINNED_VERSION="$IDL2JSON_VERSION"
 
 # Source the clap.bash file ---------------------------------------------------
 source "$SOURCE_DIR/clap.bash"
 # Define options
-clap.define short=v long=version desc="The release to install" variable=IDL2JSON_VERSION default="latest"
+clap.define short=v long=version desc="The release to install.  Options: Semantic version, latest, pinned." variable=IDL2JSON_VERSION default="latest"
```

Comment:
It seems like making `"pinned"` the default would result in fewer surprises. Shall we make "pinned" the default?
The following all have `default="pinned"`:
```
bin/dfx-software-didc-install
bin/dfx-software-didc-url
bin/dfx-software-internet-identity-ci-wasm-url
bin/dfx-software-internet-identity-install
bin/dfx-software-nns-dapp-ci-wasm-url
bin/dfx-software-sns-aggregator-ci-wasm-url
bin/dfx-software-sns-aggregator-install
```

Reply:
Yes, we can do.  Changing the default may break some code but we know most of the users!